.gitignore

```diff
@@ -33,4 +33,4 @@ app/assets/builds/
 
 .devcontainer/output
 
-TAGS
```

Comment:
This was making us ignore the `tags` views folder 🤦

Reply:
TIL .gitignore will look at any directories with a declaration made at the top level 😳